miranum-platform-showcase/stack/docker-compose.yml

```
@@ -0,0 +1,7 @@
+# Use this only in dev environments. It's not intended for production usage.
+version: '3.9'
+services:
+  camunda:
+    image: camunda/camunda-bpm-platform:7.20.0
```

```suggestion
    image: camunda/camunda-bpm-platform:7.21.0
```
Gibt schon eine neuere Version